frontend/src/i18n/locales/pt/auth.json

```diff
@@ -14,5 +14,6 @@
   "Use A Token": "Use um token",
   "Failed to get authentication information: {{ errorMessage }}": "Falha ao obter informação de autenticação: {{ errorMessage }}",
   "No permissions to list pods.": "Sem permissões para listar pods",
-  "Redirecting to main page…": "A redireccionar para a página principal…"
+  "Redirecting to main page…": "A redireccionar para a página principal…",
+  "Failed to connect. Please make sure the Kubernetes cluster is running and accessible. Error: {{ errorMessage }}": "Failed to connect. Please make sure the Kubernetes cluster is running and accessible. Error: {{ errorMessage }}"
```

Comment:
I think ones which are not translated shouldn't be English, but should stay empty.

Hopefully Joaquim can add translations for these two... maybe in this PR.

Reply:
@yolossn , this PR seems to require just the translation changes before we give a final review. Can you add that change?